lib/js/components/RichList/BasicRichListItem/BasicRichListItem.vue

```diff
@@ -71,43 +71,6 @@
 		padding: $space-xs 0;
 	}
 
-	&__eyebrow {
-		@include info-s-default-bold;
-
-		color: $color-neutral-text-weak;
-		min-width: 0;
-		overflow: hidden;
-		text-overflow: ellipsis;
-		white-space: nowrap;
-
-		&.-ds-uppercase {
-			@include info-s-extensive-bold-uppercase;
-		}
-
-		&:hover {
-			color: $color-neutral-text-weak-hovered;
-		}
-	}
-
-	&__text {
-		@include text-m-compact-bold;
-		@include invisible-scrollbar;
-
-		color: $color-neutral-text;
-		min-width: 0;
-		overflow-x: scroll;
-		white-space: nowrap;
-
-		@media #{breakpoint-s()} {
-			overflow: hidden;
-			text-overflow: ellipsis;
-		}
-
-		&:hover {
-			color: $color-neutral-text-hovered;
-		}
-	}
-
 	&.-ds-small {
 		#{$root}__content {
 			padding: $space-3xs 0;
```

Comment:
```suggestion
			padding: $space-2xs 0;
```
https://www.figma.com/design/izQdYyiBR1GQgFkaOIfIJI/LMS---DS-Components?node-id=8598-8165&m=dev